src/windowsservercore/ltsc2019/webassembly/Dockerfile

```diff
@@ -1,4 +1,19 @@
-FROM mcr.microsoft.com/dotnet-buildtools/prereqs:windowsservercore-ltsc2019-helix-amd64
+FROM mcr.microsoft.com/dotnet/framework/sdk:4.8-windowsservercore-ltsc2019
+
+SHELL ["cmd", "/S", "/C"]
+
+USER ContainerAdministrator
+
+# Install python
+RUN curl -SL --output %TEMP%\python.zip https://www.nuget.org/api/v2/package/python/3.9.5 \
+    && md C:\Python C:\PythonTemp \
+    && tar -zxf %TEMP%\python.zip -C C:\PythonTemp \
+    && xcopy /s c:\PythonTemp\tools C:\Python \
+    && copy C:\Python\python.exe C:\Python\python3.exe \
```

Comment:
Does this need to be a copy or can it be a rename?

Reply:
I still need the original file.